components/frontend/.eslintrc.json

```diff
@@ -0,0 +1,46 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "jest": true,
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:react/jsx-runtime"
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+        "no-unused-vars": [
+            "error",
+            {
+                "argsIgnorePattern": "^_"
```

Comment:
Why ignore all files starting with underscore, is it then not better to use `^_$`?

Reply:
This doesn't ignore files, it ignores function arguments that start with an underscore. For example,

```javascript
function eventHandler(_event, data) {
   // Do something with data, but ignore the event
}
```